package.json

```diff
@@ -58,13 +58,13 @@
     "minimist": "^1.1.0",
     "mkdirp": "~0.3.5",
     "moment": "^2.6.0",
-    "mongoose": "~3.8.21",
-    "mongoose-long": "^0.0.2",
+    "mongoose": "^4.13.0",
```

Comment:
Updating dependencies is a great idea, but I think it's better to stick with `~` or even "exact version" (no symbol) since `^` could possibly break something.

Reply:
You're right, I've changed to only minor versions `~`